src/hammer-vlsi/generate_properties.py

```diff
@@ -153,6 +153,18 @@ def main(args) -> int:
                                         ]
                                         )
 
+    HammerSRAMCompilerTool = Interface(module="HammerSRAMCompilerTool",
```

Comment:
I think we should try to align the name of the hammer IR name with the Python name as much as possible. In this case e.g. `sramcompiler_tool` <-> `SRAMCompilerTool`